scripts/execute.py

```diff
@@ -39,12 +40,16 @@
  imm_rates_mat, omega_S_preTP) = demog.get_pop_objs(E, S, T, min_age,
                                                     max_age, start_year,
                                                     pop_graphs)
-# imm_rates_SS = imm_rates_path[-1, :]
+
+# Get moratality rates
+totpers = E + S
+(rho_s, infmort_rate) = demog.get_mort(totpers, min_age, max_age, graph=False)
 
 # Solve the SS
 r_init = 1 / beta - 1
 xi = 0.1
-ss_params = (beta, sigma, alpha, A, delta, xi, omega_SS, imm_rates_SS, S)
+ss_params = (beta, sigma, alpha, A, delta, xi, l_tilde,
+             chi, theta, omega_SS, imm_rates_SS, rho_s, S)
 r_ss, b_sp1_ss, euler_errors_ss = SS.solve_ss(r_init, ss_params)
 
 # Economic growth
```

Comment:
I just noticed this commit here. I am going to delete my commit for `execute.py` to avoid conflict. But I have some suggestions as follows: (Not sure about time path yet; waiting to fix it after its definition)

# Solve time path
b_11 = 1.1 * b_ss
BQpath_init = np.zeros(T + S - 1)
Kpath_init = np.zeros(T + S -1)
r_path_init = np.ones(S-1)
tpi_params = (b_ss, r_11, T, S)
new_Kpath = TPI.solve_tp(g_n_path, omega_S_preTP, rho_s, imm_rates_path, tpi_params).get(new_Kpath)
new_BQpath = TPI.solve_tp(g_n_path, omega_S_preTP, rho_s, imm_rates_path, tpi_params).get(new_BQpath)

plt.plot(np.arange(S - 1), new_Kpath)
plt.title("Path of capital")
plt.show

plt.xlabel("Time", fontsize=20)
plt.ylabel("Capital", fontsize=20)
plt.savefig("capital.png")

plt.plot(np.arange(S - 1), new_BQpath)
plt.title("Path of Bequests")
plt.show

plt.xlabel("Time", fontsize=20)
plt.ylabel("Bequests", fontsize=20)
plt.savefig("BQ.png")